libsolutil/JSON.h

```diff
@@ -23,17 +23,22 @@
 
 #pragma once
 
-#include <json/json.h>
+#include <libsolutil/Assertions.h>
+#include <json/json.hpp>
 
 #include <string>
 #include <string_view>
 #include <optional>
+#include <limits>
+#include <cfloat>
+
+using Json = nlohmann::json;
```

Comment:
This is of course also not what you'd usually want to do... but I guess this makes things significantly simpler?
I guess moving this to the ```solidity`` namespace isn't enough and moving it to both ``solidity`` and ``langutil`` and such will cause issues? Well, we can just keep this, if need be.

Reply:
Ok, I said that the implementation file is bad, but not sure if this header isn't even worse, I may also have to have another look at it.

Reply:
I didn't looked into this yet, I will try to move it to `solidity::util` or just `solidityl`.

Reply:
Alright - and yeah, just ``solidity`` might actually work!

Reply:
Move it now to `solidity`.